packages/amplify-provider-awscloudformation/src/aws-utils/aws-cfn-progress-formatter.ts

```diff
@@ -1,85 +1,16 @@
-/* eslint-disable spellcheck/spell-checker */
-import columnify from 'columnify';
 import { MultiProgressBar } from 'amplify-prompts';
-
-const COLUMNIFY_WIDTH = 30;
-
-const CFN_SUCCESS_STATUS = [
-  'UPDATE_COMPLETE', 'CREATE_COMPLETE', 'DELETE_COMPLETE',
-  'DELETE_SKIPPED', 'UPDATE_ROLLBACK_COMPLETE', 'ROLLBACK_COMPLETE',
-];
-const CNF_ERROR_STATUS = [
-  'CREATE_FAILED', 'DELETE_FAILED', 'UPDATE_FAILED',
-  'UPDATE_ROLLBACK_FAILED', 'ROLLBACK_FAILED',
-];
-
-type ItemPayload = {
-  LogicalResourceId: string,
-  ResourceType: string,
-  ResourceStatus: string,
-  Timestamp: string,
-}
-
-type ProgressPayload = {
-  progressName: string,
-  envName: string
-}
-
-type EventMap = {
-  rootStackName: string,
-  envName: string,
-  projectName: string,
-  rootResources: string[],
-  eventToCategories: Map<string, string>,
-  categories: {name: string, size: number}[]
-}
-/**
- * Custom item formatter for progress bar
- */
-const createItemFormatter = (payload: ItemPayload) : { renderString: string, color: string } => {
-  let color = '';
-  const e = [{
-    logicalResourceId: payload.LogicalResourceId,
-    resourceType: payload.ResourceType,
-    resourceStatus: payload.ResourceStatus,
-    timeStamp: (new Date(payload.Timestamp)).toString(),
-  }];
-
-  const renderString = columnify(e, {
-    showHeaders: false,
-    truncate: true,
-    maxWidth: COLUMNIFY_WIDTH,
-    minWidth: COLUMNIFY_WIDTH,
-  });
-
-  if (CFN_SUCCESS_STATUS.includes(payload.ResourceStatus)) {
-    color = 'green';
-  }
-  if (CNF_ERROR_STATUS.includes(payload.ResourceStatus)) {
-    color = 'red';
-  }
-  return { renderString, color };
-};
-
-/**
- * Custom progress bar formatter
- */
-const createProgressBarFormatter = (payload : ProgressPayload,
-  value: number,
-  total: number) : string => {
-  let statusString = 'Deploying';
-  const progressNameParts = payload.progressName.split('-');
-  const name = progressNameParts.length === 1 ? progressNameParts[0] : `${progressNameParts[0]} ${progressNameParts[1]}`;
-  if (total === value) {
-    statusString = 'Deployed';
-  }
-  return `${statusString} ${name}`;
-};
+import {
+  createProgressBarFormatter,
+  createItemFormatter,
+  CFN_SUCCESS_STATUS,
+  CNF_ERROR_STATUS,
+  EventMap,
+} from '../utils/progress-bar-helpers';
 
 /**
  * Initializing the root and individual category bars
  */
-const initializeProgressBars = (eventMap : EventMap) : MultiProgressBar => {
+export const initializeProgressBars = (eventMap: EventMap): MultiProgressBar => {
```

Comment:
does this become public api of this package?

Reply:
No, it's only consumed in the aws-sdk CloudFormation helper utility in `./aws-cfn.js`